scripts/qa_crowbarsetup.sh

```diff
@@ -1525,7 +1525,7 @@ EOF
         /opt/dell/bin/json-edit -a attributes.network.networks.public.ranges.host.end -v 10.162.211.191 $netfile
     fi
     # Setup network attributes for custom MTU
-    if [[ $want_mtu_size -ne 1500 ]]; then
+    if [[ $want_mtu_size ]] && [[ $want_mtu_size -ne 1500 ]]; then
```

Comment:
@cmurphy right, that is exactly what I wanted to achieve :)
Always setting the mtu requires it to have 1. a default 2. a valid value.

Keeping the additional check is useless (wasting CPU cycles) because when we have default for the mtu size now, then it is always set and it will be written.
So just drop it and always write the mtu.